### PR TITLE
commit to filter all transactions with status unpaid

### PR DESCRIPTION
**Pull request by judeb21**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/debtor/index.blade.php [hunk 1]
```diff
@@ -2,18 +2,21 @@
 @section("custom_css")
 <link href="/backend/assets/build/css/intlTelInput.css" rel="stylesheet" type="text/css"/>
 <link rel="stylesheet" href="//cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css">
-
-    {{-- <link rel="stylesheet" href="backend/assets/css/all_users.css"> --}}
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.9.0/css/bootstrap-datepicker.css">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.17.47/css/bootstrap-datetimepicker.min.css">
```
Comment (Jeremiahiro) [inline resources/views/backend/debtor/index.blade.php:7]: how many date pickers and font awesome links are you using??

File: resources/views/backend/debtor/index.blade.php [hunk 2]
```diff
@@ -356,4 +376,21 @@
         } );
         } );
     </script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.9.0/js/bootstrap-datepicker.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/js/bootstrap.min.js"></script>
```
Comment (Jeremiahiro) [inline resources/views/backend/debtor/index.blade.php:380]: same thing here

File: resources/views/backend/debtor/index.blade.php (hunk 1 above)
Comment (judeb21) [reply resources/views/backend/debtor/index.blade.php:7]: 1 date picker and 1 time picker. The functionality requires the 2 to function

File: resources/views/backend/debtor/index.blade.php (hunk 2 above)
Comment (judeb21) [reply resources/views/backend/debtor/index.blade.php:380]: That's the bootstrap javascript function for the date picker and datetime picker. All these are required for the date picker and date time picker to work!!

**Pull request merged**